float32 v_0
float32 w_0
---
float32 C_v
float32 C_w
